tests/ui-toml/disallowed_macros/disallowed_macros.stderr

```diff
@@ -1,33 +1,11 @@
-error: use of a disallowed macro `serde::Serialize`
```

Comment:
The order here changed.  The new order is correct and matches the order that was here prior to a regressio that introduces the bug that this PR fixes.
See my first comment on the linked issue for [more details](https://github.com/rust-lang/rust-clippy/issues/14017#issuecomment-3095145917).